### ISSUE TITLE
color & font theme for NEST website

### ISSUE THREAD

**Issue by telepath37** [insightsengineering/nest#4]
Need to decide on a consistent color theme and font family for the NEST website.

Proposal: Use consistent color from NEST product logos & font

**Comment by Melkiades** [comment]
I think this is the most important aspect for the logos and website. Looking at this: [https://visme.co/blog/website-color-schemes/]( https://visme.co/blog/website-color-schemes/). I suggest Blue based ones, so 1/3/4/20/23/24/25

**Comment by Melkiades** [comment]
really like Hex Codes: #8ecae6 // #219ebc // #023047 // #ffb703 // #fb8500 ([https://venngage.com/blog/blue-color-palettes/#3](https://venngage.com/blog/blue-color-palettes/#3)

**Comment by Melkiades** [comment]
As a font I would use a monospaced one that is easier to center and it does imply different sizes for different letters. In particular I would use monoLisa: [https://crocoblock.com/blog/best-programming-fonts/](https://crocoblock.com/blog/best-programming-fonts/)

**Comment by telepath37** [comment]
> As a font I would use a monospaced one that is easier to center and it does imply different sizes for different letters. In particular I would use monoLisa: https://crocoblock.com/blog/best-programming-fonts/

Is monospace font mostly used for coding/programming? I was considering fonts like "San Francisco Pro", "Helvetica", etc.

**Comment by Melkiades** [comment]
Yes it is mainly used in programming, but I thought it was cool to use something a bit "nerdy", but I do not have a strong opinion about this. Helvetica would be cool and makes sense:
![HelveticaSpecimenCH svg](https://github.com/insightsengineering/nest/assets/11279768/16114d9f-af91-4cb4-867a-a74d36137514)

Consolas:
![consolas-regular-sample](https://github.com/insightsengineering/nest/assets/11279768/b9c2d845-5114-4d7a-a353-f2e610eedb77)

San Francisco Pro (is this the one? seems a bit appley):
![1511731670_sanfranciscofont_ru_windows-1](https://github.com/insightsengineering/nest/assets/11279768/0bccee03-a7a4-48d5-a27a-51503e8c368c)

monoLisa (very nerdy I have to admit):
![monolisa-meta](https://github.com/insightsengineering/nest/assets/11279768/8db878bd-5eb8-4654-a2f4-cb10aab20ec2)


ps from that website: Here are some things to look out for in a good programming font:
* The characters are clear and easy to read.
* There is a clear difference between the letter “O” and zero, as well as the number 1 and lowercase “L”.

**Comment by khatril** [comment]
Font suggestions: 
https://adobe-fonts.github.io/source-code-pro/

https://fonts.google.com/specimen/Roboto+Mono

**Comment by Melkiades** [comment]
I think we could still vote on this right?
